_content/news/_posts/2018-12-28_lizardxmas2018.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Lizards naar X-Mas Tournament in Kortrijk"
+date: 2018-12-11 12:00:00
+description: De Limburg Lizards doen mee aan dit internationale jeugdbasketbaltoernooi.
+cover: /news/img/LizardTraining_A.jpg
+permalink: /news/2018-12-28-lizardsxmas2018/
+---
+
+### X-mas Tournament.
+
+Van 28 tot 30 december vindt in Kortrijk het internationaal Coca Cola X-Mas Tournament plaats, met ploegen uit Europa, de V.S. en Israël.
+De Limburg Lizards zullen er met twee teams bij zijn, in deze categorieën:
+
+- Women Under 14 
+- Women Under 12 (one day tournament on 28 December)
+
+Ze bereiden zich ondertussen volop voor op deze uitdaging!
+
+Details over het toernooi vind je hier: [X-Mas Tournament](http://www.x-mastournament.be)
+
+![traininglizardsb](/news/img/LizardTraining_B.jpg)
```

Comment:
Ik vind de foto niet heel mooi, misschien vind je op de lizards facebook iets beters? 

https://www.facebook.com/pg/LimburgLizards/photos/?ref=page_internal

Reply:
Dan wachten we misschien beter op de officiële ploegfoto van Philip Wolk?

Reply:
Gaat die er komen voor deze ploegen?

Reply:
Dan best wachten ja